### PR TITLE
Add feature: company blacklist supports wildcard

### PR DESCRIPTION
**Pull request by BlackHoleExelion**
Add feature: company blacklist supports wildcard

### CODE REVIEW

**Comment by feder-cr** [comment]
@BlackHoleExelion resolve conflict

**Comment by surapuramakhil** [comment]
@BlackHoleExelion update your branch & kindly resolve Merge conflicts

**Comment by surapuramakhil** [comment]
@BlackHoleExelion you don't to close PR. just push changes to your branch

File: src/aihawk_job_manager.py [hunk 1]
```diff
@@ -513,6 +514,10 @@ def is_previously_failed_to_apply(self, link):
         file_name = "failed"
         file_path = self.output_file_directory / f"{file_name}.json"
 
+        if not file_path.exists():
```
Comment (surapuramakhil) [inline src/aihawk_job_manager.py:517]: what are these changes for?

**Pull request closed without merging**